orbs/pulumi.yml

```diff
@@ -208,3 +236,49 @@ examples:
             - pulumi/update:
                 stack: robot-co/website-prod
                 working_directory: ~/repo/src/website/
+
+  review_app_example:
+    description:
+        "Creates a new Pulumi stack and deploys the source code built as part of the workflow, running
+        tests against it. When complete, tears down the stack and cleans up any cloud resources."
+    usage:
+      version: 2.1
+      orbs:
+        pulumi: pulumi/pulumi@1.0.0
+      jobs:
+        build:
+          docker:
+            - image: circleci/node:10
+          working_directory: ~/repo/
+          steps:
+              - checkout
+
+              # Install any dependencies and build the Pulumi program.
+              - build
+
+              # Create a new Pulumi stack specifically for testing the
+              # new infrastructure changes.
+              - pulumi/login
+
+              # Create a new stack, specific to this CircleCI build,
+              # and deploy it.
+              - pulumi/stack_init:
+                  stack: robot-co/test-pr_${CIRCLE_BUILD_NUM}
+              - pulumi/update:
```

Comment:
I've fixed `pulumi/stack_update` to `pulumi/update` and also added a `stack` parameter.